Iterable/iterable-web-sdk
# Add Test Coverage for GetMessages Utilities

## JIRA Ticket(s) if any

* [MOB-3149](https://iterable.atlassian.net/browse/MOB-3149)

## Description

Adds test coverage for getMessage utility methods

## Test Steps

1. Run `yarn test`
2. Ensure tests make sense, aren't pointless, and test coverage for `srSpeak.ts` is 100% and `inapp/utils.ts` is over 90%
